site/docs/components/cookie_compliance.mdx

```diff
@@ -0,0 +1,136 @@
+(Checkbox + duration + information about the cookie)
+
+**H:** Optional cookies
+
+**B:** You can accept or decline optional cookies.
+(Checkbox + duration + information about the cookie)
+
+### UI components
+
+#### 1. Language selection
+The default UI componentes and cookie descriptions are delivered in Finnish, Swedish and English.
+
+#### 2. Cookie selection
+The cookies are divided to mandatory and optional cookies. The mandatory cookies are displayed "grayed-out" to visually support the fact that they cannot be unselected.
+
+#### Default checkboxes for cookie consent
+
+**Mandatory cookies** (visually grayed-out / disabled)
+
+- Login cookies: A mandatory cookie that allows the visitor to visit the site.
+- Language cookies: We store the interface language you have selected.
+
+**Optional cookies**
+
+- Statistic cookies: The information collected by statistical cookies is used to develop the website further.
+- Preference cookies: Preference cookies customize E.g. the look and feel of a site based on a user's choices.
+- Marketing cookies: Marketing cookies can be applied to target specific content to users.
+
+#### 3. Consent confirmation buttons
+- Accept all cookies
+- Only mandatory
+- Continue with selected (TBD)
+- Opt out (TBD)
+- Close (cross icon) -button
+
+## Usage
+
+### When to use
+ - When you need to inform users about cookie usage. 
+
+### When not to use
+ - For any other usage except informing about cookie usage.
+
+### When to use each banner type?
+| Site uses cookies              | Cookie type                    | What to do and how to inform the user?                                                                              | Example
+| ------------------------------ | ------------------------------ | ------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| Yes                            | Only mandatory cookies         | Cookie information needs to be conveyed to the user. Only requires acknowledgment and no choices from the user.     | Acknowlege -button.
+|                                | Only optional cookies          | User is given the option to choose individual cookies or accept all cookies.                                        | Cookie description(s) + duration(s) + select cookies or accept all -button.
+|                                | Mandatory and optional cookies | User is given the option to choose individual cookies, accept all cookies or only mandatory cookies.                | Cookie description(s) + duration(s) + select cookies or accept only mandatory -button.
+| No                             | –                              | **No action needed**.                                                                                               | -
+
+
+### Checklist
```

Comment:
Good stuff on this list.

To be more in line with other documentation pages, maybe this could be renamed to "Principles" since the content is very similar.

Reply:
There already is principles and the content does not suit it imo